flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/catalog/CrudExternalCatalog.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.catalog
+
+import org.apache.flink.table.api._
+
+/**
+  * The CrudExternalCatalog provides methods to create, drop, and alter (sub-)catalogs or tables.
+  *
+  * @deprecated use [[Catalog]] instead.
+  */
+@deprecated
+trait CrudExternalCatalog extends ExternalCatalog {
```

Comment:
Let's not copy those classes. I think we should either move them to common module or leave them in the flink planner and throw exception in blink-planner if they are used. For this PR I would do the latter.

Reply:
sure